Detector/DetCRD/compact/CRD_common_v01/VXD_v01_02.xml

```diff
@@ -0,0 +1,109 @@
+<lccdd>
+  <define>
+    <constant name="VXD_inner_radius" value="Vertex_inner_radius"/>
+    <constant name="VXD_outer_radius" value="Vertex_outer_radius"/>
+    <constant name="VXD_half_length"  value="Vertex_half_length"/>
+    <constant name="VXD_inner_radius_1" value="BeamPipe_VertexRegion_rmax"/>
+    <constant name="VXD_radius_r1" value="16*mm"/>
+    <constant name="VXD_radius_r3" value="37*mm"/>
+    <constant name="VXD_radius_r5" value="58*mm"/>
+    <constant name="VXD_length_r1" value="113*mm"/>
+    <constant name="VXD_length_r3" value="408*mm"/>
+    <constant name="VXD_length_r5" value="408*mm"/>
+    <constant name="VXD_cone_max_z" value="Vertex_half_length-10*mm"/>
+    <constant name="VXD_cone_min_z" value="VXD_length_r5+10*mm"/>
+  </define>
+
+  <detectors>
+    <detector id="DetID_VXD" name="VXD" type="VXD04" vis="SeeThrough" readout="VXDCollection" insideTrackingVolume="true">
+      <envelope>
+	<shape type="Assembly"/>
+	<!--shape type="BooleanShape" operation="Subtraction" material="Air" >
```

Comment:
Hi @fucd ,

Is this comment for debug only? Is that possible to remove them?

Reply:
> Hi @fucd ,
> 
> Is this comment for debug only? Is that possible to remove them?

This is another method to describe shape of envelope. But because it is complex, diffcult to fix to right, I replace it with Assembly method also supported by DD4hep. Keeping is help us to know more on DD4hep, removed is also work for the geometry. In new commit, I have removed.